python_modules/dagster/dagster/_core/definitions/decorators/decorator_assets_definition_builder.py

```diff
@@ -354,23 +371,22 @@ def from_asset_outs(
         *,
         fn: Callable[..., Any],
         op_name: str,
-        args: DecoratorAssetsDefinitionBuilderArgs,
+        asset_in_map: Mapping[str, AssetIn],
+        asset_out_map: Mapping[str, AssetOut],
+        asset_deps: Mapping[str, Set[AssetKey]],
+        upstream_asset_deps: Optional[Iterable[AssetDep]],
+        passed_args: DecoratorAssetsDefinitionBuilderArgs,
```

Comment:
Passing this through along with the other stuff feels a little weird. Maybe the check should just happen outside it?

Reply:
I agree that this is weird and lame. A bunch of stuff happening in this code path and workstream and this will go away anyways, so I'm not going to resolve it in this PR.